### ISSUE TITLE
Phrase word start and end timestamps are inconsistent between Conformer and Dragon

### ISSUE THREAD

**Issue by wolfmanstout** [talonvoice/talon#411]
The timestamps reported by phrase.words[i].start/end differ between Conformer and Dragon.  The former appears to report seconds since Talon restart, whereas the latter reports absolute milliseconds since epoch.  To reproduce:

timestamp.py
```py
from talon import Context, Module, actions, app
from talon.grammar import Phrase

mod = Module()

@mod.action_class
class Actions:
    def print_phrase_timestamps(phrase: Phrase):
        """Prints phrase start/end."""
        print("Phrase timestamps: [{}, {}]".format(phrase.words[0].start, phrase.words[-1].end))
```

timestamp.talon
```
print timestamps <phrase>: user.print_phrase_timestamps(phrase)
```

And then simply switch between Dragon and Conformer (tested on Windows with Dragon 15) and say something like "print timestamps this is a test".

Sample log from Conformer:
Phrase timestamps: [357.23526540000313, 358.3829398186078]

Sample log from Dragon:
Phrase timestamps: [1635712854725, 1635712855817]

**Comment by lunixbochs** [comment]
can you see if beta 169 is any better?

**Comment by wolfmanstout** [comment]
What I'm seeing is that both are now reporting seconds, but Dragon still reports time since epoch (and Conformer still reports time since Talon restart, or something close to that).

**Comment by lunixbochs** [comment]
Can you post an example timestamp from Dragon?

**Comment by wolfmanstout** [comment]
Sure: 1636755338.721829

Interestingly, that appears to be a timestamp for four days ago, even though I just recorded it. Based on some more tests, time seems to be advancing correctly, but the offset is wrong.

**Comment by lunixbochs** [comment]
yep just checking. I think I'll do a draconity patch for this actually.

**Comment by wolfmanstout** [comment]
Hmm, I hope this makes more sense to you than it does to me. Here's what I get when I log various timestamps in Python right after the print statement in the first post:
```
2021-11-28 15:07:32    IO Phrase timestamps: [1638077369.1700838, 1638077370.132084]
2021-11-28 15:07:32    IO time.time: 1638140852.023566, time.perf_counter: 220.2803699, time.monotonic: 63701.64
```

And here's what draconity logs at around the same time when I add the following to result_to_bson: `printf("start_time: %I64d, end_time: %I64d\n", node.start_time, node.end_time);`

start_time: 1638140849433, end_time: 1638140849874
start_time: 1638140849433, end_time: 1638140849874
start_time: 1638140849874, end_time: 1638140850576
start_time: 1638140850576, end_time: 1638140850756
start_time: 1638140849373, end_time: 1638140849874
start_time: 1638140849874, end_time: 1638140850576
start_time: 1638140850576, end_time: 1638140850756
start_time: 1638140850756, end_time: 1638140850916
start_time: 1638140850916, end_time: 1638140850977
start_time: 1638140850977, end_time: 1638140851538

All is from the utterance "print timestamps this is a test".

**Comment by lunixbochs** [comment]
should be fixed in 0.2.0-257, please close Dragon before updating

**Comment by wolfmanstout** [comment]
Awesome, confirmed that this is fixed!
